rust/main/hyperlane-base/src/contract_sync/mod.rs

```diff
@@ -222,8 +222,15 @@ where
                 );
 
                 if let Some(tx) = self.broadcast_sender.as_ref() {
+                    // Deduplicate txids in logs to only send unique txids
```

Comment:
nit - would slightly reword, since it may be hard to understand what's happening without more context: "If multiple logs occur in the same transaction they'll have the same `transaction_id`. Deduplicate their txids to avoid doing wasteful queries in txid indexer"